### PR TITLE
Support keys wrapped in dbl quotes

### PR DESCRIPTION
**Pull request by rothrock**
Issue: 1729

Trim off any leading or lagging dbl quotes.

I could see where this could break if someone created 
a tag key like:
 "\"host\"": "server01",

### CODE REVIEW

**Comment by rothrock** [comment]
Oh, and the httpd test fails. I don't know why. Any ideas?

**Comment by rothrock** [comment]
Works with curl, though:
```bash
curl -G 'http://localhost:8086/query' --data-urlencode "db=toast" --data-urlencode 'q=SHOW TAG VALUES WITH KEY = "host"'

{"results":[{"series":[{"name":"hostTagValues","columns":["host"],"values":[["server01"]]}]}]}
```

**Comment by dgnorton** [comment]
lgtm  :shipit:

File: influxql/parser_test.go [hunk 1]
```diff
@@ -293,6 +293,19 @@ func TestParser_ParseStatement(t *testing.T) {
 			},
 		},
 
+		// SHOW TAG VALUES WITH KEY = "..."
+		{
+			s: `SHOW TAG VALUES WITH KEY = "host" WHERE region = 'uswest'`,
+			stmt: &influxql.ShowTagValuesStatement{
+				TagKeys: []string{`"host"`},
```
Comment (pauldix) [inline influxql/parser_test.go]: why is the parser yielding the quotes? I would expect the parser to take care of clearing those out. Anywhere an identifier is yielded it should be cleared of quotes.

Comment (dgnorton) [inline influxql/parser_test.go]: We may be able to refactor the scanner / parser code to get rid of this.  I'd have to look at it.

**Pull request merged**